front/lib/auth.ts

```diff
@@ -249,12 +264,15 @@ export class Authenticator {
           name: this._workspace.name,
           allowedDomain: this._workspace.allowedDomain || null,
           role: this._role,
-          plan: planForWorkspace(this._workspace),
           upgradedAt: this._workspace.upgradedAt?.getTime() || null,
         }
       : null;
   }
 
+  plan(): PlanType | null {
```

Comment:
Should we make it async? Do you already know if you want it to be async eventually or not?

Reply:
I did not make it async, it is done here on the next PR: https://github.com/dust-tt/dust/pull/2245